core/src/main/resources/hudson/model/Computer/_scriptText.jelly

```diff
@@ -25,6 +25,6 @@ THE SOFTWARE.
 <!--
   Called from doScriptText() to display the execution result.
 -->
-<?jelly escape-by-default='false'?>
+<?jelly escape-by-default='true'?>
 <st:compress xmlns:j="jelly:core" xmlns:st="jelly:stapler">
-<st:contentType value="text/plain;charset=UTF-8" />${output}</st:compress>
\ No newline at end of file
+<st:contentType value="text/plain;charset=UTF-8" /><st:out value="${output}"/></st:compress>
```

Comment:
Please review this usage of `st:out`

Reply:
It sets a content type header, so if the result isn't escaped, then it's fine.

Reply:
`st:out` escape the `<` and `&`.
`j:out` just render the value without escaping.